### PR TITLE
docs: Move fractional evaluation pages to website

### PR DESCRIPTION
**Pull request by agardnerIT**
## This PR
- Moves fractional evaluation information to website

### CODE REVIEW

**Review by beeme1mr — CHANGES_REQUESTED** [review]

File: web-docs/reference/fractional_evaluation.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+In this case, `25%` of the email addresses will receive `red`, `25%` will receive `blue`, and so on.
+
+Importantly, the evaluations are "sticky" meaning that the same `email` address will always belong to the same "bucket" and thus always receive the same color.
+
+## Fractional Evaluation: Technical Description
+
+The `fractionalEvaluation` operation is a custom JsonLogic operation which deterministically selects a variant based on
+the defined distribution of each variant (as a percentage).
+This works by hashing ([xxHash](https://cyan4973.github.io/xxHash/))
```
Comment (craigpastro) [inline web-docs/reference/fractional_evaluation.md:17]: 👋 I guess this line should be updated as #846 has been merged. https://github.com/open-feature/flagd/blob/c3c9e4e40aeae7e75b1b9ab13bb9a40264be84e5/docs/configuration/fractional_evaluation.md?plain=1#L17

**Comment by beeme1mr** [comment]
@toddbaert @craigpastro, now that we've released the updated fractional evaluation logic, could you please confirm these docs are accurate?

**Comment by toddbaert** [comment]
> @toddbaert @craigpastro, now that we've released the updated fractional evaluation logic, could you please confirm these docs are accurate?

They are not at all accurate because of the conflict. The "old" contents are being moved, and instead the "new" contents should be, I guess, but I'm not sure I fully understand the intent of the move itself.

**Comment by craigpastro** [comment]
All examples and samples are still using the now deprecated `fractionalEvaluation` key. Should those be updated?

**Comment by toddbaert** [comment]
> All examples and samples are still using the now deprecated `fractionalEvaluation` key. Should those be updated?

Ya, basically this "move" should be performed again with the updated content. I dont think any content needs to be changed.

**Comment by agardnerIT** [comment]
Closing in favour of one larger, but easier to review PR: #825

**Pull request closed without merging**